src/elements/fields/PhoneField/index.tsx

```diff
@@ -197,42 +199,44 @@ function PhoneField({
             '&:hover': { backgroundColor: '#e6e6e633' }
           }}
           ref={triggerRef}
-          onClick={() => setShow(!show)}
+          onClick={(e: any) => {
+            // TODO: position should be set staticly based on the input, not on the click. but refs were all null?
+            setMenuPosition({
```

Comment:
small change to make here. I forgot that I commented out `ref={triggerRef}` 🙈 when I was trying to do this originally, so pretty sure that was my issue